compiler/next/lib/resolution/resolution-types.cpp

```diff
@@ -17,11 +17,235 @@
  * limitations under the License.
  */
 
+#include "chpl/queries/update-functions.h"
 #include "chpl/resolution/resolution-types.h"
+#include "chpl/uast/Builder.h"
+#include "chpl/uast/Formal.h"
 
 namespace chpl {
 namespace resolution {
 
+using namespace uast;
+
+void ResolutionResultByPostorderID::setupForSymbol(const ASTNode* ast) {
+  assert(Builder::astTagIndicatesNewIdScope(ast->tag()));
+  vec.resize(ast->id().numContainedChildren());
```

Comment:
Is it safe to assume `vec` will never have something already in it, so we don't need to worry about accidentally overwriting because we failed to account for there already being contents?

Reply:
Well, if there were previous contents, the index of those contents would have to be < the number of contained children, otherwise it would not be a result by postorder ID. So, the resize will leave any thing that already existed alone and just append empty slots.

All that said, I expect that every time this is called, `vec` is empty.